gmso/core/topology.py

```diff
@@ -965,6 +966,171 @@ def get_index(self, member):
 
         return index
 
+    def to_datatables(self, parameter="sites", labels=None, unyts_bool=True):
+        """Return a pandas dataframe object for the sites in a topology
+
+        Parameters
+        ----------
+        parameter : a string
+            A string determining what aspects of the gmso topology will be reported. Options are: 'sites','bonds','angles'
+        and 'dihedrals'. Defaults to 'sites'.
+        labels : List of strings that are attributes of the topology site.
+            Examples of these can be found by printing `dir(topology.sites[0])`
```

Comment:
```suggestion
            Examples of these can be found by printing `topology.sites[0].__dict__`
```
May also add a link/reference to the documentation of `Sites`

Reply:
also, are these labels only for `sites` and not other connection objects? (e.g. case where `parameter='bond'`, will the labels be attributes of bonds?)

Reply:
So there are no unique attributes of a bond besides for within the bond_expression. I'm not certain if gmso supports adding other labels, but I thought the only important things to print would be the type expression information, which is a default feature. The additional labels just export the site information of the sites within the connection_members in case you are interested in that information additionally. Which is why I direct the user only to that list of options for the labels.